docs/topics/absolute_beginners.rst

```diff
@@ -0,0 +1,103 @@
+.. _absolute-beginners:
+
+#############################################
+Conceptual introduction for absolute beginner
+#############################################
+
+We assume that most people interested in OpenPathSampling are familiar with
+the basic tools of molecular simulation: molecular dynamics and Monte Carlo
+sampling. [#MC]_ However, you might be new to path sampling methods.
+Furthermore, OpenPathSampling uses a rather novel perspective on path
+sampling in general. This introduction is aimed at users who are either new
+to path sampling or who want to have a better understanding of the
+conceptual framework that OpenPathSampling is built on.
+
+Path sampling techniques apply, obviously, to "paths." We will use the word
+"trajectory" interchangeably with "path." Each path is made up of several
+"snapshots" or "frames." Again, we often use these words interchangeably.
+
+Collective variables are functions of a snapshot
+================================================
+
+For a system with :math:`N` atoms, each snapshot on a trajectory (in 3D
```

Comment:
We might explain the use of _snapshot_ as a frozen picture of the system state at a particular in time (which is more or can be more than just a phase space point, etc...)